settings.php

```diff
@@ -975,38 +976,69 @@ function convertseconds($argument)
                                                     when enabling DNSSEC. A DNSSEC resolver test can be found
                                                     <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
                                                 </div>
-                                                <strong>Conditional Forwarding</strong>
-                                                <p>If not configured as your DHCP server, Pi-hole won't be able to
+                                                <p>Validate DNS replies and cache DNSSEC data. When forwarding DNS
+                                                   queries, Pi-hole requests the DNSSEC records needed to validate
+                                                   the replies. If a domain fails validation or the upstream does not
+                                                   support DNSSEC, this setting can cause issues resolving domains.
+                                                   Use Google, Cloudflare, DNS.WATCH, Quad9, or another DNS
+                                                   server which supports DNSSEC when activating DNSSEC. Note that
+                                                   the size of your log might increase significantly
+                                                   when enabling DNSSEC. A DNSSEC resolver test can be found
+                                                   <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
+                                                <label>Conditional forwarding</label>
```

Comment:
Maybe using an `h3` or `h4` would be better. There's not a form here.

Reply:
Thanks, addressed.